### PR TITLE
sdBAL-Mint-and-Stake

### PR DESCRIPTION
**Pull request by Zen-Maxi**
Transaction uses all 80BAL-20wETH BPT from payroll to approve stake dao BalancerDepositor as a spender, then depositAll is called on the contract with lock and stake set to true to earn excess rewards. Once our position is created we have the option to delegate our position and boost if it is sensible.

### CODE REVIEW

File: MaxiOps/Payroll_Txns/Q4_2024/sdBAL-Mint-and-Stake.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        "inputs": [
+          { "internalType": "address", "name": "spender", "type": "address" },
+          { "internalType": "uint256", "name": "amount", "type": "uint256" }
+        ],
+        "name": "approve",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "spender": "0x3e0d44542972859de3CAdaF856B1a4FD351B4D2E",
+        "amount": "8033000000000000000000"
```
Comment (gosuto-inzasheru) [inline MaxiOps/Payroll_Txns/Q4_2024/sdBAL-Mint-and-Stake.json:28]:
```suggestion
        "amount": "8032855070070031668947"
```

**Review by gosuto-inzasheru — COMMENTED** [review]
- [sdbal gauge token](https://etherscan.io/address/0x3e8c72655e48591d93e6dfda16823db0ff23d859) appears to be transferrable ✅
- did we also want to do some general review of the sdbal contract?

**Comment by Zen-Maxi** [comment]
@gosuto-inzasheru wdym by a review of sdBAL contract? this is not the vester or anything new. Just vanilla staking whole sdBAL bag into stake daos normal interface to earn yields until we can switch to the vesters.

**Review by gosuto-inzasheru — APPROVED** [review]
good to go then

**Pull request merged**